### DISCUSSION TITLE
Add tracks to the selected one

### DISCUSSION THREAD

**Discussion by Llullcobain** [helgoboss/helgobox#891] [Ideas]
Hi! I'm really happy with the use of realearn, but I got myself stuck trying to do something that i'm not sure it's possible.
So, I have two buttons to navigate through tracks (on select the track on the left, on select the track on the right). Is there a way to Add the next channel to the selection (so create a gang) like you do when you do Ctrl + Click?? 
I don't manage to find the way with Browse track or select/unselect options.

Thanks!
Raul Uberos

**Comment by helgoboss** [comment]
This is currently not possible. But would be not difficult to add. I'm going to convert this into a feature request (idea).
